test/index.html

```diff
@@ -9,6 +9,10 @@
   <body>
     <script>
       WCT.loadSuites([
+        'hc-nested-list_test.html',
```

Comment:
The default is whatever that browser uses, so for chrome and FF it's shadow, for IE and safari it's shady still (I believe).  Since the goal is to move all browsers over to shadow exclusively we want to ensure it's always tested against shadow.